x/poolmanager/types/hooks.go

```diff
@@ -0,0 +1,24 @@
+package types
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+type PoolManagerHooks interface {
+	// AfterPoolCreated is called after CreatePool
+	AfterPoolCreated(ctx sdk.Context, sender sdk.AccAddress, poolId uint64)
+}
```

Comment:
We should not need this. There are already `PoolCreationListeners` in `listeners.go` that we can use instead